databroker/src/grpc/server.rs

```diff
@@ -138,23 +131,55 @@ where
     .await
 }
 
-pub async fn serve_with_incoming_shutdown<F>(
-    listener: TcpListener,
+pub async fn serve_uds<F>(
```

Comment:
Do we want to get rid of `uds` everywhere, i.e. rename this one to `serve_unix_domain_socket` or similar?